R/html_document.R

```diff
@@ -323,6 +326,13 @@ html_document <- function(toc = FALSE,
                                  list(html_dependency_pagedtable()))
   }
 
+  # anchor-sections
+  if (anchor_sections) {
+    extra_dependencies <- append(extra_dependencies,
+                                 list(html_dependency_anchor_sections()))
+    lua_filters <- pkg_file_lua("anchor-sections.lua")
```

Comment:
Do you intend to do that ?
```suggestion
    lua_filters <- pkg_file_lua(lua_filters, "anchor-sections.lua")
```
As of now, it would overwritten the value defined line 209
https://github.com/rstudio/rmarkdown/pull/1884/files#diff-d244a3f1d96cb8bba005d659d4cdc69dR209

Reply:
Obviously it is to be protective regarding future change, because if we want to add a filter before that, we would need to think to add this concatenation. 

I prefer we be take measure in advance as it is done usually for args
https://github.com/rstudio/rmarkdown/blob/832db9142a57e3083a5f46c9d20a88afc4a9cc4c/R/word_document.R#L56-L59

Reply:
Oops. You're right.
I intended the `args`'s way.
I'll make a fix.

Reply:
fixed via efcd03ea